sechub-scan/src/main/resources/templates/report/html/scanresult.html

```diff
@@ -112,31 +390,65 @@ <h2 class='redFindingHeadline'>Red findings</h2>
-                                         <div class="rTableHead" style="width:5%">Line</div>
-                                         <div class="rTableHead" style="width:5%">Column</div>
-                                         <div class="rTableHead" style="width:10%">Relevant part</div>
-                                         <div class="rTableHead" style="width:25%">Source</div>
-                                     </div>
-                                     <div class="rTableRow" th:each="codeScanEntry : ${codeScanEntries.get(finding.id)}">
-                                         <div class="rTableCell" th:text="${codeScanEntry.location}" style="width:25%">Location</div>
-                                         <div class="rTableCell" th:text="${codeScanEntry.line}" style="width:5%">12</div>
-                                         <div class="rTableCell" th:text="${codeScanEntry.column}" style="width:5%">33</div>
-                                         <div class="rTableCell" th:text="${codeScanEntry.relevantPart}" style="width:10%">Source</div>
-                                         <div class="rTableCell" th:text="${codeScanEntry.source}" style="width:25%">Source</div>
-                                     </div>
-                                </div>
+                                
+                                <div class="rTable" th:if="${codeScanSupport.isCodeScan(finding)}">
+								    	<div class="rTableRow">
+								        	<div class="rTableHead" style="width:5%">Id</div>
+								            <div class="rTableHead" style="width:25%">Location</div>
+								            <div class="rTableHead" style="width:5%">Line</div>
+								            <div class="rTableHead" style="width:5%">Column</div>
+								            <div class="rTableHead" style="width:10%">Relevant part</div>
+								            <div class="rTableHead" style="width:25%">Source</div>
+								        </div>
+       									<div class="rTableRow" th:if="${codeScanEntries.get(finding.id).size() - 1  == iterator.index}" th:each="codeScanEntry,iterator : ${codeScanEntries.get(finding.id)}">
+											<div class="rTableCell" th:text="${iterator.index}" style="width:5%">Id</div>
+											<div class="rTableCell" th:text="${codeScanEntry.location}" style="width:25%">Location</div>
+											<div class="rTableCell" th:text="${codeScanEntry.line}" style="width:5%">Line</div>
+											<div class="rTableCell" th:text="${codeScanEntry.column}" style="width:5%">Column</div>
+											<div class="rTableCell" th:text="${codeScanEntry.relevantPart}" style="width:10%">Relevant Part</div>
+											<div class="rTableCell" th:text="${codeScanEntry.source}" style="width:25%">Source</div>
+										</div>
+								</div>
+							    	         
+							   	<br />
+							   	
+                                <details th:if="${codeScanSupport.isCodeScan(finding)}">
```

Comment:
This and the following lines are just a repetition of the same code from red findings. Is there a way to use a Thymeleafe fragment or something instead of repeating the same code?

Reply:
moved complete findings-cell construction to fragment.